tests/Smoke/InstallViaComposerTest.php

```diff
@@ -164,8 +157,55 @@ public function testInstallationViaArtifactIsPossible(): void
         self::assertCommandsWork($stepsToPrepareArtifact, $tmpArtifactPath);
         self::assertCommandsWork($this->stepsToVerifyInstallation, $tmpPath);
 
-        $fs->remove($tmpPath);
-        $fs->remove($tmpArtifactPath);
+        $this->fs->remove($tmpPath);
+        $this->fs->remove($tmpArtifactPath);
+    }
+
+    public function testDoctrineAnnotationRulesetThrowsAnExceptionWhenDoctrinePackagesAreNotInstalled(): void
+    {
+        $tmpPath = $this->createFakeComposerProject($this->currentCodeAsComposerDependency);
+
+        self::assertCommandsWork(
+            [
+                'composer install -q',
+                'composer dump-autoload --optimize',
+                'php vendor/autoload.php',
+                'echo "<?php class /** @SomeAnnotation() */Foo {}" > test.php',
```

Comment:
```suggestion
                'echo "<?php /** @SomeAnnotation() */ class Foo {}" > test.php',
```
That's the correct Doctrine's annotation

Reply:
It doesn't matter, it's only here to trigger fixer with `/**` 😉. But for the sake of purity 👌.

Reply:
It does matter, at current `master`:
```sh
kub@:~/code/friendsofphp/PHP-CS-Fixer(master)$ echo "<?php class /** @SomeAnnotation() */Foo {}" > test.php
kub@:~/code/friendsofphp/PHP-CS-Fixer(master)$ php php-cs-fixer fix --diff --dry-run -vvv --rules=@DoctrineAnnotation test.php
PHP CS Fixer 3.23.1-DEV First Steps by Fabien Potencier and Dariusz Ruminski.
PHP runtime: 8.2.9
Loaded config default from "/home/kuba/code/friendsofphp/PHP-CS-Fixer/.php-cs-fixer.dist.php".
Paths from configuration file have been overridden by paths provided as command arguments.
.                                                                                                                     1 / 1 (100%)
Legend: .-no changes, F-fixed, S-skipped (cached or empty file), I-invalid file syntax (file ignored), E-error

Found 0 of 1 files that can be fixed in 0.003 seconds, 14.000 MB memory used
kub@:~/code/friendsofphp/PHP-CS-Fixer(master)$ echo "<?php /** @SomeAnnotation() */ class Foo {}" > test.php
kub@:~/code/friendsofphp/PHP-CS-Fixer(master)$ php php-cs-fixer fix --diff --dry-run -vvv --rules=@DoctrineAnnotation test.php
PHP CS Fixer 3.23.1-DEV First Steps by Fabien Potencier and Dariusz Ruminski.
PHP runtime: 8.2.9
Loaded config default from "/home/kuba/code/friendsofphp/PHP-CS-Fixer/.php-cs-fixer.dist.php".
Paths from configuration file have been overridden by paths provided as command arguments.
F                                                                                                                     1 / 1 (100%)
Legend: .-no changes, F-fixed, S-skipped (cached or empty file), I-invalid file syntax (file ignored), E-error
   1) test.php (doctrine_annotation_braces)
      ---------- begin diff ----------
--- /home/kuba/code/friendsofphp/PHP-CS-Fixer/test.php
+++ /home/kuba/code/friendsofphp/PHP-CS-Fixer/test.php
@@ -1 +1 @@
-<?php /** @SomeAnnotation() */ class Foo {}
+<?php /** @SomeAnnotation */ class Foo {}

      ----------- end diff -----------


Found 1 of 1 files that can be fixed in 0.005 seconds, 14.000 MB memory used
```

Reply:
OK, it does not matter in case of this PR, because it's not about fixing or not, but only triggering the fixer that throws an exception or not (depending if required dependencies are installed or not). Anyway, I'll fix it.